features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -189,6 +189,8 @@
         <RenewRefreshTokenForRefreshGrant>true</RenewRefreshTokenForRefreshGrant>
         <!-- Process the token before storing it in database, e.g. encrypting -->
         <TokenPersistenceProcessor>org.wso2.carbon.identity.oauth.tokenprocessor.PlainTextPersistenceProcessor</TokenPersistenceProcessor>
+        <!-- Get hashing oauth keys (consumer secret, access token, refresh token and authorization code) configuration is enabled or not -->
```

Comment:
Please enhance the description of the config, by referring a example like https://github.com/Biruntha/carbon-identity-framework/blob/7e3ccc7fcb89fefff60360ba293da680146e2c68/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml#L293:L305